integration-tests/suites/repeated_network_flow.go

```diff
@@ -88,7 +88,28 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	numIter := strconv.Itoa(s.NumIter)
 	sleepBetweenCurlTime := strconv.Itoa(s.SleepBetweenCurlTime)
 	sleepBetweenIterations := strconv.Itoa(s.SleepBetweenIterations)
-	_, err = s.execContainer("nginx-curl", []string{"/usr/bin/schedule-curls.sh", numMetaIter, numIter, sleepBetweenCurlTime, sleepBetweenIterations, serverAddress})
+	_, err = s.execContainerShellScript("nginx-curl", `
+		#!/usr/bin/env sh
```

Comment:
Maybe save this to a variable. Also I don't know if this is the best place for it. Maybe it is.